### PR TITLE
First version of the Danish translations

### PR DESCRIPTION
**Pull request by cryptobench**
Let me know if something is wrong and needs to be fixed. 

Probs to @ErCiccione for guiding me through.

### CODE REVIEW

File: lang/languages.xml [hunk 1]
```diff
@@ -38,4 +38,5 @@ List of available languages for your wallet's seed:
         <language display_name="עברית" locale="he_HE" wallet_language="English" flag="/lang/flags/israel.png" qs="none"/>
         <language display_name="한국어" locale="ko_KO" wallet_language="English" flag="/lang/flags/south_korea.png" qs="none"/>
         <language display_name="Română" locale="ro_RO" wallet_language="English" flag="/lang/flags/romania.png" qs="none"/>
-</languages>
+        <language display_name="Dansk" locale="da_DK" wallet-language="English" flag="/Laang/flags/denmark.png" qs="none"/>
+        </languages>
```
[the remark this answers is not in the document]
Comment (cryptobench) [reply lang/languages.xml]: Okay, I think I updated everything now. Do I have to delete this pull or is it updated automatically??

File: translations/monero-core_da.ts [hunk 2]
```diff
@@ -0,0 +1,2148 @@
...
+    <name>AddressBookTable</name>
+    <message>
+        <location filename="../components/AddressBookTable.qml" line="47"/>
+        <source>No more results</source>
+        <translation>Ikke flere resultater</translation>
+    </message>
+    <message>
+        <location filename="../components/AddressBookTable.qml" line="104"/>
+        <source>Payment ID:</source>
+        <translation>Betalings ID</translation>
```
Comment (erciccione) [inline translations/monero-core_da.ts]: Needs `:` after `Betalings ID`

File: translations/monero-core_da.ts [hunk 3]
```diff
@@ -0,0 +1,2148 @@
...
+    </message>
+</context>
+<context>
+    <name>BasicPanel</name>
+    <message>
+        <location filename="../BasicPanel.qml" line="97"/>
+        <source>Locked Balance:</source>
+        <translation>Låst Saldo</translation>
```
Comment (erciccione) [inline translations/monero-core_da.ts]: missing `:`

File: translations/monero-core_da.ts [hunk 4]
```diff
@@ -0,0 +1,2148 @@
...
+    </message>
+    <message>
+        <location filename="../BasicPanel.qml" line="110"/>
+        <source>78.9239845</source>
+        <translation>78.9239845</translation>
+    </message>
+    <message>
+        <location filename="../BasicPanel.qml" line="133"/>
+        <source>Available Balance:</source>
+        <translation>Tilgængelig Saldo</translation>
```
Comment (erciccione) [inline translations/monero-core_da.ts]: missing `:`

File: translations/monero-core_da.ts [hunk 5]
```diff
@@ -0,0 +1,2148 @@
...
+    <message>
+        <location filename="../pages/Transfer.qml" line="571"/>
+        <location filename="../pages/Transfer.qml" line="632"/>
+        <source>Please choose a file</source>
+        <translation>Vælg venligst en fil</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="583"/>
+        <source>Can&apos;t load unsigned transaction: </source>
+        <translation>Kan ikke loade usignerede transaktioner</translation>
```
Comment (erciccione) [inline translations/monero-core_da.ts]: missing `: ` (with a space after `:`)

File: translations/monero-core_da.ts [hunk 6]
```diff
@@ -0,0 +1,2148 @@
...
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="611"/>
+        <source>Confirmation</source>
+        <translation>Bekræftelser</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="639"/>
+        <source>Can&apos;t submit transaction: </source>
+        <translation>Kan ikke indsende transaktion</translation>
```
Comment (erciccione) [inline translations/monero-core_da.ts]: missing `: ` (with a space after `:`)

File: translations/monero-core_da.ts [hunk 7]
```diff
@@ -0,0 +1,2148 @@
...
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="105"/>
+        <source> - the secret transaction key supplied by the sender</source>
+        <translation> - den hemmelige transaktions nøgle suppleret af senderen</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="110"/>
+        <source>If a payment had several transactions then each must be checked and the results combined.</source>
+        <translation>hvis en betaling havde flere transaktioner så må hver eneste blive checket og resultaterne lagt sammen</translation>
```
Comment (erciccione) [inline translations/monero-core_da.ts]: Missing `.` at the end

File: translations/monero-core_da.ts [hunk 8]
```diff
@@ -0,0 +1,2148 @@
...
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="123"/>
+        <source>Address</source>
+        <translation>Adresse</translation>
+    </message>
+    <message>
+        <location filenam
```
Comment (erciccione) [inline translations/monero-core_da.ts]: Missing `.` at the end

Comment (erciccione) [inline translations/monero-core_da.ts]: Missing `.` at the end

Comment (erciccione) [inline translations/monero-core_da.ts]: Missing `.` at the end

Comment (erciccione) [inline translations/monero-core_da.ts]: Missing `.` at the end

Comment (erciccione) [inline translations/monero-core_da.ts]: missing `:` and space

Comment (erciccione) [inline translations/monero-core_da.ts]: missing `:`

Comment (cryptobench) [reply translations/monero-core_da.ts]: Errrrrr, little confused on this one, where i'm missing something. Quadruple checked this one

Comment (cryptobench) [reply translations/monero-core_da.ts]: ergg confused. 6 of those are for lines 1533-1536 with different errors

Comment (erciccione) [reply translations/monero-core_da.ts]: Don't know what happened here, please ignore this and the following 6 comments of this review

File: translations/monero-core_da.ts [hunk 9]
```diff
@@ -0,0 +1,2148 @@
...
+        <source>If a payment had several transactions then each must be checked and the results combined.</source>
+        <translation>hvis en betaling havde flere transaktioner så må hver eneste blive checket og resultaterne lagt sammen.</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="123"/>
+        <source>Address</source>
+        <translation>Adresse</translation>
+    </message>
+    <message>
+        <location
```
Comment (erciccione) [inline translations/monero-core_da.ts]: missing `.` at the end

Comment (erciccione) [inline translations/monero-core_da.ts]: missing `.` at the end

Comment (erciccione) [inline translations/monero-core_da.ts]: missing `.` at the end

Comment (erciccione) [inline translations/monero-core_da.ts]: missing `.` at the end

Comment (erciccione) [inline translations/monero-core_da.ts]: missing `:` and space at the end

Comment (erciccione) [inline translations/monero-core_da.ts]: missing `:` at the end

File: lang/languages.xml [hunk 10]
```diff
@@ -38,4 +38,5 @@ List of available languages for your wallet's seed:
         <language display_name="עברית" locale="he_HE" wallet_language="English" flag="/lang/flags/israel.png" qs="none"/>
         <language display_name="한국어" locale="ko_KO" wallet_language="English" flag="/lang/flags/south_korea.png" qs="none"/>
         <language display_name="Română" locale="ro_RO" wallet_language="English" flag="/lang/flags/romania.png" qs="none"/>
-</languages>
+        <language display_name="Dansk" locale="da_DK" wallet-language="English" flag="/Laang/flags/denmark.png" qs="none"/>
```
Comment (erciccione) [inline lang/languages.xml]: `/Laang/flags/denmark.png` should be `lang/flags/denmark.png`

Comment (cryptobench) [reply lang/languages.xml]: WOops. Will fix immediately. :-)

**Pull request merged**